### DISCUSSION TITLE
Share documents

### DISCUSSION THREAD

**Discussion by sgarciahelguera** [Cubeseed/Backend#74] [Ideas]
In the meeting I said that adding a wiki page with a link to a google doc would work, but maybe here is  better ? 
@Reid-T-W what do you think? 
@Cubeseed/everybody  ?

**Comment by sgarciahelguera** [comment]
Not a lot of context there, I am talking about sharing documents that are not plain text, like google docs.

**Reply by Reid-T-W** [reply]
Yes, this would be better, I'll share the link here.

**Reply by abu-hasib** [reply]
@sgarciahelguera you tagged everyone.

**Reply by sgarciahelguera** [reply]
Yes, intentionally.

**Comment by Reid-T-W** [comment]
@sgarciahelguera here it is 

[Initial Setup required to get the AWS S3 storage working](https://docs.google.com/document/d/1VNk0Doz34FrtaCWjHqgOk6_F3kqsVFR-Tl2nzEcsFEY/edit?usp=sharing)
